src/server/binary/binaryServerStarter.ts

```diff
@@ -206,13 +206,32 @@ function getBinaryEnvironment(): any {
   return process.env;
 }
 
+/**
+ * Returns the platform of the current operating system.
+ * If the OS is Alpine Linux, it returns 'alpine'.
+ * Otherwise, it returns the platform string provided by the 'os' module.
+ *
+ * @returns The platform of the current operating system.
+ */
+function getPlatform (): ReturnType<typeof os.platform> | 'alpine' {
+  if (fs.existsSync('/etc/os-release')) {
```

Comment:
Would it be sufficient to just use the presence of `/etc/alpine-release` ? to return `alpine` ? I'd like to keep this as minimal as possible without any reading.

Reply:
thanks, I didn't know that file exists. updated